x-pack/plugins/security_solution/public/timelines/components/side_panel/index.tsx

```diff
@@ -36,85 +36,83 @@ interface DetailsPanelProps {
  * To prevent duplication the `isFlyoutView` prop is passed to determine the layout that should be used
  * `tabType` defaults to query and `handleOnPanelClosed` defaults to unsetting the default query tab which is used for the flyout panel
  */
-export const DetailsPanel = React.memo(
-  ({
-    browserFields,
-    docValueFields,
-    handleOnPanelClosed,
-    isFlyoutView,
-    tabType,
-    timelineId,
-  }: DetailsPanelProps) => {
-    const dispatch = useDispatch();
-    const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
-    const expandedDetail = useDeepEqualSelector((state) => {
-      return (getTimeline(state, timelineId) ?? timelineDefaults).expandedDetail;
-    });
+export const DetailsPanel = ({
```

Comment:
The only change in this file was removing `React.memo`. It may be causing an issue where the `event_details` view isn't switching properly. The props to this component do not change when the `expanded_detail` changes, and as this is just a wrapper of memoized components, it's unnecessary.